nebula-components/components/_modal.scss

```diff
@@ -57,7 +57,7 @@
     right: 0;
     width: 40px;
     height: 40px;
-    background: $nb-color-black;
-    color: $nb-color-white;
+    background-color: $nb-color-alpha;
```

Comment:
Have changed to black, added in a title attr and have put some styling in to also accommodate for some text within this button. See here: 

![screen shot 2017-08-04 at 13 49 21](https://user-images.githubusercontent.com/11405108/28969317-0f12c1a8-791c-11e7-98ef-ff8e6d7e8d47.png)

![screen shot 2017-08-04 at 13 52 07](https://user-images.githubusercontent.com/11405108/28969353-3074b8ec-791c-11e7-8e54-dc1718e32b3c.png)